randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -257,8 +256,13 @@ Extra - room_id: [6]
   * Layers: default
   > Door to Sciser Stall
       Any of the following:
-          # Destroy bomb blocks by standing off of crab: https://youtu.be/h8Q6TKHg8kM
-          Hi-Jump and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies and Can Use Bombs and Can Use Springball
+          All of the following:
+              # Destroy bomb blocks by standing off of crab: https://youtu.be/h8Q6TKHg8kM
+              Stand On Frozen Enemies (Beginner) and Can Freeze Enemies and Can Use Bombs
+              Any of the following:
+                  Hi-Jump
+                  # NHJ: https://youtu.be/_F80ASGzrvo
+                  Movement (Advanced) and Wall Jump (Beginner)
```

Comment:
I would just make it movement beginner and use the ledge grab video: https://youtu.be/6RhuTs1mD1c

Reply:
imo, using a ledge grab to circumvent a wall jump trick requirement doesn't sound right to me.

Reply:
I guess my only other worry is that the timer for the crab to thaw is pretty tight. and imo doing ledge grabs is a lot slower than regular wall jumps

Reply:
ledge grabs arent really that much slower.